docs/deployment-examples/custom-scripts/powershell-core.md

```diff
@@ -0,0 +1,23 @@
+---
+title: PowerShell Core
+description: Enabling PowerShell Core.
+position: 109
+---
+
+In Octopus Version 2019.9.0 we released support for PowerShell Core. When executing a PowerShell script in Octopus as a part of a step, you can now specify whether Octopus should use PowerShell or PowerShell Core. This feature will allow you to execute your PowerShell scripts on Linux targets with PowerShell Core installed. For information on how this affects proxy, [see here](/docs/infrastructure/deployment-targets/proxy-support.md#powershell-core-scripts).
+
+You can enable the option to select PowerShell Core from the Configure Features button on a step in your deployment.
+
+![custom feature](images/customfeature.png)
+
+Enabling this feature will give you the option to select PowerShell Core to execute your PowerShell scripts. By default, Octopus will execute your PowerShell scripts with standard PowerShell.
+
+![powershellcore](images/powershellcore.png)
+
+## PowerShell Core on Linux Targets
+
+You will first need to ensure that you add the PowerShell directory to your `$PATH` before Octopus can use it.
+
+For Linux targets with PowerShell Core installed, you can use this feature to directly run your PowerShell scripts against those targets. However, there are some things to keep in mind here.
+
+If your package contains both `.ps1` & `.sh` scripts and you are calling PowerShell Core on a non-Windows target, Octopus will attempt to execute both scripts. This can cause some unexpected behavior depending on the scrip, so ensure that your package only contains a single copy of your deploy scripts if you are using PowerShell Core.
```

Comment:
```suggestion
If your package contains both `.ps1` & `.sh` scripts and you are deploying to a non-Windows target, Octopus will attempt to execute both scripts. This may cause some unexpected behavior depending on the script, so ensure that your package only contains a single copy of your deployment scripts.
```